components/dashboard/dashboard.go

```diff
@@ -139,6 +139,27 @@ func (d *Dashboard) ReconcileComponent(owner metav1.Object, cli client.Client, s
 		if err != nil {
 			return fmt.Errorf("failed to set dashboard ISV from %s: %v", PathISVSM, err)
 		}
+		// ConsoleLink handling
+		consoleRoute := &routev1.Route{}
+		err = cli.Get(context.TODO(), client.ObjectKey{Name: "console", Namespace: "openshift-console"}, consoleRoute)
+		if err != nil && !apierrs.IsNotFound(err) {
+			return fmt.Errorf("Error getting console route URL : %v", err)
+		}
+		domainIndex := strings.Index(consoleRoute.Spec.Host, ".")
```

Comment:
Wouldn't this cause an error if console route  is not found?

Reply:
yes, but we have `componentErrorList` and will continue to next component, right?
as for the dashboard itself, call to `DeployManifestsFromPath` are ahead of this part, should the configs should be applied